src/registrar/assets/js/uswds-edited.js

```diff
@@ -6128,7 +6158,13 @@ const setUpAttributes = tooltipTrigger => {
   tooltipBody.setAttribute("aria-hidden", "true");
 
   // place the text in the tooltip
-  tooltipBody.textContent = tooltipContent;
+  // DOTGOV: nest the text element to allow us creater control over width and wrapping behavior
+   tooltipBody.innerHTML = `
+    <div class="usa-tooltip__content">
```

Comment:
The div inside a span does not seem to trigger accessibility check errors, but it goes counter to HTML specifications. A div is a block level container, a span is an inline-level container span[https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span]

CSS-wise, you can easily make a span behave like a div in the layout by setting display: block on it. 

Q: Is there something else that's keeping you from using a span with a display block style on?

Reply:
I see your point, and I get nesting divs inside spans is generally a no-no.   However, the element is set to display:block already when it becomes visible (.usa-tooltip__body.is-set), yet there is something else going on that positions the tooltip incorrectly above the tooltip icon when we use `<span>`.  We might want to huddle on this.